cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenGiving.js

```diff
@@ -0,0 +1,42 @@
+var adyenGivingNode = document.getElementById("donate-container");
+
+function handleOnDonate(state, component) {
+    if(state.isValid){
+        var selectedAmount = state.data.amount;
+        var donationData = {
+            amountValue: selectedAmount.value,
+            amountCurrency: selectedAmount.currency,
+            orderNo: orderNo,
+            pspReference: pspReference
+        };
+
+        $.ajax({
+            url: 'Adyen-Donate',
+            type: 'post',
+            data: donationData,
+            success: function () {
+                component.setStatus("success");
+            }
+        });
+    }
+}
+
+function handleOnCancel(state, component) {
+    $("#adyenGiving").slideUp();
```

Comment:
If possible let's not add more jquery to the codebase.
Can you try it with
  var adyenGiving = document. getElementById("adyenGiving");
  adyenGiving.style.transition = "all 1s ease-in-out";
  adyenGiving.style.height = "0px";